src/Tests/Contract/OrderHelperTrait.php

```diff
@@ -416,4 +418,143 @@ private function createOrderWithDiscount(Ids $ids): Order
             'id' => Ids::get('order-discount-id'),
         ]);
     }
+
+    private function createOrderForApplication(IdsCollection $ids): Order
```

Comment:
can't we merge this together with `createOrder`?

Reply:
No, we need this, because we need `real` acceptable data for Braintree.
Maybe we could use some faker library, like foundry